src/commands/transaction/multisigmodification.ts

```diff
@@ -54,7 +54,7 @@ export class CommandOptions extends AnnounceAggregateTransactionsOptions {
 
     @option({
         flag: 'p',
-        description: 'Cosignatory account public key.',
+        description: 'Cosignatory account public key (separated by a comma).',
         validator: new PublicKeyValidator(),
```

Comment:
todo: create``PublicKeysValidator`` to support multiple keys validation.

Reply:
fixed